spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -23,6 +23,12 @@
       expect(json['steam']).to have_key("url")
       expect(json['steam']).to have_key("nickname")
     end
+
+    it 'returns 404 if user does not exist' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.